packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -83,6 +83,13 @@ function get_prompt_from_diffusers_model(model: ModelData): string | undefined {
 	}
 }
 
+export const ben2 = (model: ModelData): string[] => [
+	`from ben2 import AutoModel
+
+model = AutoModel.from_pretrained("${model.id}")
```

Comment:
Since this model is very specific, maybe you'd like to provide a complete example for background removal ? Something like:

```py
import requests
from PIL import Image

url = "https://upload.wikimedia.org/wikipedia/commons/b/b6/Felis_catus-cat_on_snow.jpg"
image = Image.open(requests.get(url, stream=True).raw)

... # init model + run inference
```